frontend/src/features/recommendation/SearchFilter.spec.tsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { ThemeProvider } from '@material-ui/core/styles';
+import { theme } from 'src/theme';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+
+import SearchFilter from './SearchFilter';
+import { dateChoices } from './DateFilter';
+import { languageChoices } from './LanguageFilter';
+import { mainCriteriaNames } from 'src/utils/constants';
+
+describe('Filters feature', () => {
```

Comment:
tests durations :
  Filters feature
    ✓ Can open and close the filters menu (328 ms)
    ✓ Check that all the filter checkboxes and sliders are present (197 ms)
    ✓ Can only check one date filter at once (409 ms)
    ✓ Can check multiple language filters at once (290 ms)